docs/website/docs/plus/features/projects.md

```diff
@@ -0,0 +1,194 @@
+  
+```yaml
+datasets:
+    github_events_dataset:
+        destination:
+          - duckdb
+```
+
+#### Cache 🧪
+
+In this section you specify the input table(s) that you want to transform, and the output table(s) that you want to write after performing the transformations. The example below loads the table "events" from the destination dataset "github_events_dataset" into a local cache, then transforms it using the transformations inside the `transformations/` folder, and finally writes two tables back into the dataset "github_events_dataset": the original "events" table, and the transformed "events_aggregated" table. Read more about how local cache is used for transformations [here](../core-concepts/datasets.md).
+  
+The cache feature is currently limited to specific use cases and is only compatible with data stored in filesystem-based destinations. Please, make sure that the input dataset for the cache is located in the filesystem-based destination ([Iceberg](../ecosystem/iceberg.md), [Delta](../ecosystem/delta.md), or [Cloud storage and filesystem](../../dlt-ecosystem/destinations/filesystem.md)).
+
+```yaml
+caches:
+  github_events_cache:
+    inputs:
+      - dataset: github_events_dataset
+        tables:
+          events: events
+    outputs:
+      - dataset: github_events_dataset
+        tables:
+          events: events
+          events_aggregated: events_aggregated
+```
+:::note
+🚧 This feature is under development. Interested in becoming an early tester? [Join dlt+ early access](https://info.dlthub.com/waiting-list)
+:::
+
+#### Transformations 🧪
+
+Here you specify the settings for your transformations. In the code example we define an arrow-based transformation that will operate on the cache "github_events_cache". It will make use of code in the `transformations/` folder. Read more about how transformations are done [here](../features/transformations/index.md). 
+
+```yaml
+transformations:
+  github_events_transformations:
+    engine: arrow
+    cache: github_events_cache
+```
+:::note
+🚧 This feature is under development. Interested in becoming an early tester? [Join dlt+ early access](https://info.dlthub.com/waiting-list)
+:::
+
+#### Profiles
+
+You can use the profiles section to define different environments (example: dev, staging, prod, tests). One package may have multiple profiles which can be specified using dlt+ cli commands. The default profile name is `dev`. It's created automatically alongside with `tests` profile.
+
+```yaml
+profiles:
+    dev: # Using "dev" profile will write to local filesystem
+        destinations:
+            delta_lake:
+                type: filesystem
+                bucket_url: ${tmp_dir}delta_lake
+    prod: # Using "prod" profle will write to an s3 bucktet
+        destinations:
+            delta_lake:
+                type: filesystem
+                bucket_url: s3://dlt-ci-test-bucket/dlt_example_project/
+
```

Comment:
```suggestion
profiles:
  dev: # Using "dev" profile will write to local filesystem
    destinations:
      delta_lake:
        type: filesystem
        bucket_url: ${tmp_dir}delta_lake
  prod: # Using "prod" profle will write to an s3 bucktet
    destinations:
      delta_lake:
        type: filesystem
        bucket_url: s3://dlt-ci-test-bucket/dlt_example_project/ 
```

Reply:
fixed in https://github.com/dlt-hub/dlt/pull/2325